pdns/dnsdistdist/docs/guides/serverselection.rst

```diff
@@ -46,10 +46,19 @@ The current hash algorithm is based on the qname of the query.
 ``chashed``
 ~~~~~~~~~~~
 
+.. versionadded: 1.3.3
+
 ``chashed`` is a consistent hashing distribution policy. Identical questions with identical hashes will be distributed to the same servers. But unlike the ``whashed`` policy, this distribution will keep consistent over time. Adding or removing servers will only remap a small part of the queries.
 
+Increasing the weight of servers to a value larger than the default is required to get a good distribution of queries. Small values like 100 or 1000 should be enough to get a correct distribution.
```

Comment:
I was wondering if it might help to cap weight to a min value (100?) when using consistent hashing-based distribution ?